Initial commit

AndrejSH1981 opened the pull request.

AndrejSH1981 committed:
Initial commit

ArthurKharatian commented on src/lab1/lab1git.java:
```diff
@@ -0,0 +1,66 @@
+        // 3.
+        int number = 10500;
+        double result = 0;
+        result = (number/10)/10;
```
здесь теряется остаток. подставьте 3 вместо number

AndrejSH1981 committed:
Update lab1git.java

//3. Подставил "d" в выражение "result = (number/10)/10",
получилось "result = (number/10d)/10d" для получения остатка в результате

AndrejSH1981 committed:
Delete src/labs directory

AndrejSH1981 commented:
Сделал исправления в выражении, добавил "d": // 3.
        int number = 10500;
        double result = 0;
        result = (number/10d)/10d;
